playbooks/common/openshift-cluster/upgrades/init.yml

```diff
@@ -23,3 +23,16 @@
     set_fact:
       os_firewall_use_firewalld: false
     when: "'Active: active' in service_iptables_status.stdout"
+
+- name: Verify Host Requirements
+  hosts: OSEv3
+  roles:
+  - openshift_health_checker
+  vars:
+  - r_openshift_health_checker_playbook_context: "upgrade"
```

Comment:
Presumably, we need to handle this in https://github.com/openshift/openshift-ansible/blob/2d4709b5a65c5bb2777277420ee08f8554efb1ab/roles/openshift_health_checker/callback_plugins/zz_failure_summary.py#L98? /cc @sosiouxme